Source/WebCore/css/calc/CSSCalcTree+Simplification.cpp

```diff
@@ -1279,6 +1279,20 @@ std::optional<Child> simplify(Sign& root, const SimplificationOptions& options)
     );
 }
 
+std::optional<Child> simplify(Anchor& anchor, const SimplificationOptions& options)
+{
+    if (!options.conversionData || !options.conversionData->style())
+        return { };
+
+    // FIMXE: Evaluate the anchor.
```

Comment:
Typo: FIMXE -> FIXME.

Reply:
anchor-calc [~/webkit/OpenSource] git grep FIMXE Source | wc
      13     155    1590

🏛️

Reply:
Well I'll be!